packages/devtools_app/lib/src/config_specific/import_export/import_export.dart

```diff
@@ -104,8 +105,7 @@ abstract class ExportController {
     DateTime? time,
   }) {
     time ??= DateTime.now();
-    final timestamp =
-        '${time.year}_${time.month}_${time.day}-${time.microsecondsSinceEpoch}';
+    final timestamp = DateFormat('yy-MM-dd_HH:mm:ss.SSS').format(time);
```

Comment:
what about MM-DD-YY ? I would think this would be the most human readable format

Reply:
1. Europe uses DD-MM-YY, so, if you do not know format upfront, you cannot read it
2. We want the files to be sortable by time